src/products/views/ProductUpdate/useChannelVariantListings.ts

```diff
@@ -0,0 +1,51 @@
+import { ChannelData } from "@saleor/channels/utils";
+import useStateFromProps from "@saleor/hooks/useStateFromProps";
+import { arrayDiff } from "@saleor/utils/arrays";
+import isEqual from "lodash/isEqual";
+import { useMemo } from "react";
+
+import { ChannelsWithVariantsData } from "./types";
+import { createFromChannels } from "./utils";
+
+function useChannelVariantListings(channels: ChannelData[]) {
```

Comment:
Why `function` instead of `() => ...`?

Reply:
Call me old fasioned, but I'm used to write `function` when declaring module-scope functions